### ISSUE TITLE
[Question] Exactly how are the included models trained?

### ISSUE THREAD

**Issue by QuintinPope** [QData/TextAttack#296]
When I use the pretrained models (from e.g., --model roberta-base-imdb), what are the exact training parameters of this model (epochs, batch size, learning rate and scheduler, training data, early stopping, augmentations, etc)?

Thanks for any information you can provide. Also, thanks for writing this codebase. It's very convenient!

**Comment by jxmorris12** [comment]
Hey @QuintinPope -- that's a great question. We should make this information more accessible.

For each of our transformers, we selected the best out of a grid search over a bunch of possible hyperparameters. So it actually varies from model to model.

All of our models have model cards on the HuggingFace model hub. So for now, the easiest way to figure this out is:
1. Go to our page on the model hub: https://huggingface.co/textattack
2. Find the model you're looking for and select its page: https://huggingface.co/textattack/roberta-base-imdb
3. Scroll down to the model card

Here's what the one you asked about looks like:

![Screen Shot 2020-10-08 at 10 21 24 AM](https://user-images.githubusercontent.com/13238952/95471702-37046280-0950-11eb-957b-2e413da0caf7.png)

Let me know if you have any questions or notice something missing!

**Comment by qiyanjun** [comment]
@QuintinPope  we added model full details in https://textattack.readthedocs.io/en/latest/3recipes/models.html
